### PR TITLE
Use spoof check off for connectivity tests

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: pkg/util/network/network.go [hunk 1]
```diff
@@ -27,6 +27,8 @@ func CreateSriovNetwork(clientSet *testclient.ClientSet, name string, namespace
 			ResourceName:     resourceName,
 			IPAM:             ipam,
 			NetworkNamespace: namespace,
+			// We need this to be able to run the connectivity checks on mellanox cards
```
Comment (fedepaol) [inline pkg/util/network/network.go]: Can we do this only for mellanox?

**Comment by fedepaol** [comment]
/lgtm
Feel free to remove it once you validate this

**Comment by SchSeba** [comment]
/hold cancel

Validated.

**Pull request merged**